tests/test-cases/Group1-Docker-Commands/1-4-Docker-Create.robot

```diff
@@ -33,6 +33,21 @@ Create with volume
     #Should Be Equal As Integers  ${rc}  0
     #Should Not Contain  ${output}  Error
 
+Create with named volume
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} create -v test-named-vol:/testdir busybox
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} start ${output}
+    Should Be Equal As Integers  ${rc}  0
+    Should Not Contain  ${output}  Error
+
+    ${status}=  Get State Of Github Issue  1718
```

Comment:
The problem with warn is that it doesn't really get any attention. We have something like 40 or 50 warns on any ci run. I think it is important to have a fail in order actually get attention paid. If 1718 isnt exactly a perfect fit, maybe create a new issue specifically for it?